package.json

```diff
@@ -1,54 +1,37 @@
 {
-  "name": "@probot/serverless-lambda",
+  "name": "@probot/adapter-aws-lambda-serverless",
```

Comment:
npm question: Does this mean the old name will disappear from registry? I mean, `@probot/serverless-lambda@0.x.x` won't exist anymore?

Reply:
not disappear, but it will be deprecated